packages/react-core/src/demos/ComposableMenu/examples/ComposableApplicationLauncher.tsx

```diff
@@ -250,12 +250,7 @@ export const ComposableApplicationLauncher: React.FunctionComponent = () => {
     // eslint-disable-next-line no-console
     <Menu ref={menuRef} onActionClick={onFavorite} onSelect={(_ev, itemId) => console.log('selected', itemId)}>
       <MenuInput>
-        <TextInput
-          aria-label="Filter menu items"
-          iconVariant="search"
-          type="search"
-          onChange={value => onTextChange(value)}
-        />
+        <SearchInput aria-label="Filter menu items" type="search" onChange={value => onTextChange(value)} />
```

Comment:
@tlabaj - any reason SearchInput type isn't defaulted to search?

Reply:
It looks like when I added it in that I matched the default `type` of TextInputGroupMain which was `text`. It probably could be defaulted to `search` instead though.

Reply:
@tlabaj I actually have the following PR open in Core that relates to the "x" icon for search input types 
https://github.com/patternfly/patternfly/pull/5211